samples/boards/nrf/battery/README.rst

```diff
@@ -11,29 +11,40 @@ infrastructure to measure the voltage of the device power supply.  Two
 power supply configurations are supported:
 
 * If the board devicetree has a ``/vbatt`` node with compatible
-  ``voltage-divider`` then the voltage is measured using that divider.
-* Otherwise the power source is assumed to be directly connected to the
-  digital voltage signal, and the internal source for ``Vdd`` is
-  selected.
-
-An example of a devicetree node describing a voltage divider for battery
-monitoring is:
-
-.. code-block:: none
-
-   / {
-   	vbatt {
-   		compatible = "voltage-divider";
-   		io-channels = <&adc 4>;
-   		output-ohms = <180000>;
-   		full-ohms = <(1500000 + 180000)>;
-   		power-gpios = <&sx1509b 4 0>;
-   	};
-   };
+  ``voltage-divider`` then the voltage is measured using that divider. An
+  example of a devicetree node describing a voltage divider for battery
+  monitoring is:
+
+   .. code-block:: devicetree
+
+      / {
+         vbatt {
+            compatible = "voltage-divider";
+            io-channels = <&adc 4>;
+            output-ohms = <180000>;
+            full-ohms = <(1500000 + 180000)>;
+            power-gpios = <&sx1509b 4 0>;
+         };
+      };
+
+* If the board does not have a voltage divider and so no ``/vbatt`` node
+  present, the ADC configuration (device and channel) needs to be provided via
+  the ``zephyr,user`` node. In this case the power source is assumed to be
+  directly connected to the digital voltage signal, and the internal source for
+  ``Vdd`` is selected. An example of a Devicetree configuration for this case is
+  shown below:
+
+   .. code-block :: devicetree
+
+      / {
+         zephyr,user {
+            io-channels = <&adc 4>;
```

Comment:
Do we need to add an example like this in the .dts of an actual board?

Reply:
I can't find a suitable nRF board, and to be honest, I think that a divider will be the most common case way of measurements (to avoid exceeding hw range limit).